recipes/ncbi-vdb/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2.9.0" %}
+{% set sha256 = "b4099e2fc3349eaf487219fbe798b22124949c89ffa1e7e6fbaa73a5178c8aff" %}
+
+package:
+  name: ncbi-vdb
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  fn: ncbi-vdb-{{ version }}.tar.gz
+  url: https://github.com/ncbi/ncbi-vdb/archive/{{ version }}-1.tar.gz
+  sha256: {{ sha256 }}
+
+
+requirements:
+  build:
+    - gcc # [not osx]
+    - llvm # [osx]
+    - perl
+    - libiconv
+    - libxml2
+    - ncbi-ngs-sdk
+    - bison
+
+  run:
+    #
+
+about:
+  home: https://github.com/ncbi/ncbi-vdb
+  license: Public Domain
+  summary: The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives.
+
+test:
+  commands:
+    - test -e ${PREFIX}/lib64/libncbi-vdb.so # [not osx]
+    - test -e ${PREFIX}/lib64/libncbi-vdb.dylib # [osx]
+
```

Comment:
I still would like to see some binary using the library executed here. Very likely that'll show that the `run:` section needs some content, too.